cmd/lakectl/cmd/branch_protect.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/treeverse/lakefs/pkg/api"
+)
+
+const (
+	branchProtectAddCmdArgs = 2
+)
+
+var branchProtectCmd = &cobra.Command{
+	Use:   "branch-protect",
+	Short: "Create and manage branch protection rules",
+	Long:  "Define branch protection rules to prevent direct changes. Changes to protected branches can only be done by merging from other branches.",
+}
+var branchProtectListCmd = &cobra.Command{
+	Use:     "list <repo uri>",
+	Short:   "List all branch protection rules",
+	Example: "list lakefs://<repository>",
```

Comment:
Use a full example. I don't think it concatenates.
Here and elsewhere.

Reply:
Done